File: app/src/main/java/pl/przybylo/flights/dto/FlightViewDto.java
package pl.przybylo.flights.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FlightViewDto {

    private Long id;
    private String name;
    private PlaceWithDateDto departure;
    private PlaceWithDateDto arrival;
    private Integer sitCount;
    private Integer touristCount;
    private BigDecimal ticketPrice;
    private List<TouristViewDto> tourists;

}

File: web/src/main/app/project/src/app/zapytanie/model/flight-view.dto.ts
import {PlaceWithDateDto} from "./place-with-date.dto";
import {TouristViewDto} from "./tourist-view.dto";

export interface FlightViewDto {
    id: number;
    name: string;
    departure: PlaceWithDateDto;
    arrival: PlaceWithDateDto;
    sitCount: number;
    touristCount: number;
    ticketPrice: number;
    tourists: Array<TouristViewDto>;
}
